src/App.js

```diff
@@ -0,0 +1,24 @@
+import Nav from './components/Nav';
+// import '../src/App.css';
```

Comment:
- [ ] **Required:** Kindly remove all unused and commented code from your codebase. It is considered best practice and professional.